new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java

```diff
@@ -1546,10 +1739,20 @@ private HtmlPage submitConfigureForm(
     fileField.setContentType("multipart/form-data");
 
     // Now submit the form by clicking the button and get back the second page.
-    HtmlPage page2 = button.click();
-    assertNotNull(page2, "can't reach page after submit");
-    assertFalse((page2.asText()).contains("Error 500--Internal Server Error"),
-            "page returns Error 500--Internal Server Error");
+    HtmlPage page2 = null;
+    try {
+      page2 = button.click();
+      assertNotNull(page2, "can't reach page after submit");
+      assertFalse((page2.asText()).contains("Error 500--Internal Server Error"),
+          "page returns Error 500--Internal Server Error");
+    } catch (ClassCastException ex) {
+      logger.info(" Can't generate html page, collecting the error ");
+      TextPage page3 = button.click();
+      assertNotNull(page3, "can't reach page after submit");
+      assertTrue(page3.getContent().contains("Unable to contact the REST API"),
+          "submit does not return html page, here is received page "
+          + page3.getContent());
+    }
     // wait time to update configuration
     Thread.sleep(15 * 1000);
```

Comment:
Any other way to eliminate hard-coded 15s sleep

Reply:
no, prometheus is configured to push metrics every 10 secs, so we need to wait 15 until new metric will be pushed